components/account/AccountUserName.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import styles from "@/styles/components/account/account-user-name.module.scss";
+
+export default function AccountUserName() {
+	const [isEditActive, setIsEditActive] = useState(false);
+	const [userName, setUserName] = useState("Lio Giladi");
```

Comment:
where is "Lio Giladi" coming from? The default value should come from external source -> should come from a prop

Reply:
noted